README.md

```diff
@@ -38,32 +38,33 @@ A `dev-plugin-settings.json` file is also needed in SciGateway's `micro-frontend
-
 ### Specifying environment variables
 
-For local development it is recommended you adjust the URLs for the `REST_API` and `DATA_VIEWER` in [`.env`](https://github.com/fiaisis/frontend/blob/main/.env) to point to the staging `API` and `data viewer`.
+Unless you have a working API and data viewer set-up locally you will want the frontend to point to the ones in staging which require your machine to be on the company VPN. The URLs for the `REST_API` and `DATA_VIEWER` are found in [`.env`](https://github.com/fiaisis/frontend/blob/main/.env).
 
-## Available scripts
+## Running the frontend for the first time
 
-In the project directory, you can run:
+Assuming all the previous steps have been completed, you can now run the frontend.
 
 ### `yarn install`
 
-Installs the necessary dependencies for the project. You will need to run this command before running the project for the first time or after adding new dependencies to the project.
+Installs the necessary dependencies for the project. You will also need to run this after adding new dependencies or switching to branches with a modified [`package.json`](https://github.com/fiaisis/frontend/blob/main/package.json) file.
+
+### `yarn build`
+
+Builds the app for production to the `build` folder. It correctly bundles React in production mode and optimizes the build for the best performance. The build is minified and the filenames include the hashes. You will need to build every time you make changes you want to test.
 
 ### `yarn start`
 
-Runs the app in the development mode. Open [http://localhost:3000/fia](http://localhost:3000/fia) to view it in the browser.
+You can now open a terminal in SciGateway and have it act as a parent application for running the frontend. You only need to run `yarn start` and the FIA frontend will be running on [http://localhost:3000/fia](http://localhost:3000/fia).
 
-The page will reload if you make edits. You will also see any lint errors in the console.
+## Container files
 
-### `yarn build`
+Certain features of the frontend such as the help page are handled by files in SciGateway which are overwritten during production to display the correct information to users. Files for this purpose are stored in the [`container`](https://github.com/fiaisis/frontend/tree/main/container) folder. Any changes made locally to this folder won't be visible when running the web application -- so if you want to test changes you you will need to run a [Docker](https://www.docker.com/) container by building [`frontend.dockerfile`](https://github.com/fiaisis/frontend/blob/main/container/scigateway.dockerfile) and [`scigateway.dockerfile`](https://github.com/fiaisis/frontend/blob/main/container/scigateway.dockerfile) then viewing the changes in a browser.
```

Comment:
Would it be possible to replace this with the following (in the context menu you can select edit on my comment to copy the whole thing including formatting):

Certain features of the frontend such as the help page are handled by files in SciGateway which are overwritten during production to display the correct information to users. Files for this purpose are stored in the [`container`](https://github.com/fiaisis/frontend/tree/main/container) folder. Any changes made locally to this folder won't be visible when running the web application using yarn start, and are only available when running in a container -- so if you want to test changes you need to create a new image, to do this run (you will need [Docker](https://www.docker.com/) installed for these commands):
Frontend:
```bash
docker build . -t ghcr.io/fiaisis/frontend -f ./frontend.dockerfile  
```
Scigateway (note the working directory needs to be in the container folder for this):
```bash
docker build . -t ghcr.io/fiaisis/scigateway -f ./scigateway.dockerfile
```
Then you can run the containers using these commands:
Frontend:
```bash
docker run --rm -it -p 8080:80 ghcr.io/fiaisis/frontend
```
SciGateway (note, without the frontend running on the same local network this will not display the plugin):
```bash
docker run --rm -it -p 8080:80 ghcr.io/fiaisis/scigateway
```
To access the websites made by the above containers navigate to http://localhost:8080